lfs/lfs.go

```diff
@@ -29,6 +30,7 @@ var (
 	LocalGitStorageDir string // parent of objects/lfs (may be same as LocalGitDir but may not)
 	LocalMediaDir      string // root of lfs objects
 	LocalObjectTempDir string // where temporarily downloading objects are stored
```

Comment:
Agreed, this PR was intended to be just a first step. If this PR is on the right track, we can continue with the refactoring.